versions.props

```diff
@@ -12,7 +12,7 @@ com.googlecode.json-simple:json-simple = 1.1.1
 com.jayway.awaitility:awaitility = 1.6.5
 com.palantir.config.crypto:encrypted-config-value-module = 1.0.0
 com.palantir.remoting:* = 0.13.0
-com.palantir.remoting1:* = 1.0.3
+com.palantir.remoting1:* = 1.1.0
```

Comment:
technically this is a breaking change as 1.1.0 includes `[BREAKING] Clients support TLS 1.2 only (could break applications talking to servers that provide TLS <1.2 only)`, see https://github.com/palantir/http-remoting/releases

If we're bumping to 1.1.x, should probably bump to latest `1.1.2`